Kubernetes/flannel/l2bridge/cni/config/cni.conf

```diff
@@ -1,45 +1,45 @@
-                                                "Value":  {
-                                                              "Type":  "OutBoundNAT",
-                                                              "ExceptionList":  [
-                                                                                    "192.168.0.0/16",
-                                                                                    "11.0.0.0/8",
-                                                                                    "10.124.24.0/23"
-                                                                                ]
-                                                          }
-                                            },
-                                            {
-                                                "Name":  "EndpointPolicy",
-                                                "Value":  {
-                                                              "Type":  "ROUTE",
-                                                              "DestinationPrefix":  "11.0.0.0/8",
-                                                              "NeedEncap":  true
-                                                          }
-                                            },
-                                            {
-                                                "Name":  "EndpointPolicy",
-                                                "Value":  {
-                                                              "Type":  "ROUTE",
-                                                              "DestinationPrefix":  "10.124.24.33/32",
-                                                              "NeedEncap":  true
-                                                          }
-                                            }
-                                        ]
-                 }
-}
+{
+    "cniVersion":  "0.2.0",
+    "name":  "cbr0",
+    "type":  "flannel",
+    "delegate":  {
+                     "type":  "l2bridge",
+                     "dns":  {
+                                 "Nameservers":  [
+                                                     "10.96.0.10"
+                                                 ],
+                                 "Search":  [
+                                                "svc.cluster.local"
+                                            ]
+                             },
+                     "AdditionalArgs":  [
+                                            {
+                                                "Name":  "EndpointPolicy",
+                                                "Value":  {
+                                                              "Type":  "OutBoundNAT",
+                                                              "ExceptionList":  [
+                                                                                    "10.244.0.0/16",

```

Comment:
These should all be configurable or made clear (e.g. "<Cluster CIDR>", "<Management CIDR>", etc.) that user's need to enter in their Cluster CIDR, Service VIP CIDR, and Management Network CIDR ranges here. They will most likely need to change from these default values.

Reply:
These are configurable and will all be overwritten by the script. This is done once by specify them in the start.ps1 script:
`.\start.ps1 -ManagementIP <Windows_Worker_Mgmt_IP> -ClusterCIDR <ClusterCIDR> -ServiceCIDR <SvcCIDR> -KubeDnsServiceIP <KubeDNSIP>`